e2e/playwright/actions/src/tests/create-library.spec.ts

```diff
@@ -0,0 +1,263 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { Utils, ApiClientFactory, getUserState, test, libraryErrors } from '@alfresco/playwright-shared';
+import { SiteBodyCreate } from '@alfresco/js-api';
+
+test.use({ storageState: getUserState('hruser') });
+test.describe('Create Libraries ', () => {
+  const apiClientFactory = new ApiClientFactory();
+  let randomLibraryName: string;
+  let randomLibraryId: string;
+  let randomLibraryDescription: string;
+  const libraryDialogTitle = 'Create Library';
+  const libraryNameLebel = 'Name *';
```

Comment:
I think there is a typo, should be `libraryNameLabel`

Reply:
change provided in [a9a9a98](https://github.com/Alfresco/alfresco-content-app/pull/3318/commits/a9a9a98483d2b0cba804fbecd1a3f2d79a8345a6)